packages/strapi-icons/package.json

```
@@ -29,8 +29,9 @@
     "react-dom": "^17.0.2"
   },
   "scripts": {
-    "build": "yarn generate:icons && vite build && yarn generate:types",
-    "generate:icons": "svgr --typescript --icon -d ./src ./assets/icons",
+    "build": "yarn generate:icons && yarn generate:vite:build && yarn generate:types",
+    "generate:vite:build": "vite build",
```

```suggestion
    "build:vite": "vite build",
```

IMO not necessary since its such a short command, but it's definitely not a generate script 😅 

or just `build:prod` which is what i do sometimes...

---

I went for `build:prod` because that is how it is setup in the DS package.